applovin_max/ios/Classes/AppLovinMAXAdViewWidget.m

```diff
@@ -84,6 +73,23 @@ - (void)setLocalExtraParameters:(NSDictionary<NSString *, id> *)parameterDict
     }
 }
 
+- (void)setAutoRefreshEnabled:(BOOL)autoRefresh
+{
+    if ( autoRefresh )
+    {
+        [self.adView startAutoRefresh];
+    }
+    else
+    {
+        [self.adView stopAutoRefresh];
+    }
+}
+
+-(BOOL)isAutoRefreshEnabled
```

Comment:
Now I realized - is the getter never used? Like, before your latest change, we only set the value but never query for it. If so, we should actually take a different approach of not using properties and explicitly declaring `- (void)setAutoRefreshEnabled:(BOOL)autoRefresh`.

Reply:
Please review it again.  If I took your message correctly, we wouldn't need the properties.